### DISCUSSION TITLE
Saving an Article

### DISCUSSION THREAD

**Discussion by SockworkOrange** [dailydotdev/daily#644] [Feature Requests]
While I understand why I can't submit an article until I have enough reputation, if daily.dev intends to be a consolidated place of tech reading - perhaps it can be possible to add a bookmark of an article that isn't in daily.dev? Without submitting it, I mean.

**Comment by idoshamun** [comment]
Hi @SockworkOrange,

Thanks for the suggestion. There's an open discussion on this matter: #629. Let's keep the discussion going there
